Tools/scripts/checkpyc.py

```diff
@@ -54,7 +54,7 @@ def main():
                 mtime = get_long(mtime_str)
                 if mtime in {0, -1}:
                     print('Bad ".pyc" file', repr(name_c))
-                elif mtime != st[ST_MTIME]:
```

Comment:
I don't know if this code should be modified, but it is modified: should we also cast to int()?

Reply:
like that?
```python
elif int(mtime) != int(st[ST_MTIME] & 0xFFFF_FFFF)
```

Reply:
`mtime` is already int. But `st[ST_MTIME]` is float.